modules/machineset-migrating-compute-nodes-to-new-sd-rhv.adoc

```diff
@@ -0,0 +1,94 @@
+
+[id="machineset-creating-vm-template-rhv"]
+== Creating a virtual machine template on {rh-virtualization}
+
+Before you can create a new virtual machine template you must first create a new temporary virtual machine from the old template.
+
+.Procedure
+
+. Identify the virtual machine template:
++
+[source,terminal]
+----
+$ oc get -o jsonpath='{.spec.template.spec.providerSpec.value.template_name}{"\n"}' machineset
+----
+. Create a new virtual machine in the {rh-virtualization-engine-name}.
+.. Click menu:Compute[Virtual Machines]
+.. Click the btn:[New] button.
+. Select the template you identified above and leave all other settings unchanged. You do not have to start the virtual machine.
+. When the virtual machine is created, select it and click *More Actions*{kebab}, then click *Make template*. The *Make template* dialog opens.
+. In the *Make template* dialog, enter the template name and select the desired storage domain under *Target*.
+. Delete the temporary virtual machine.
+
+.Additional resources
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Creating_a_Virtual_Machine_Based_on_a_Template[Creating a Virtual Machine Based on a Template] in the Red Hat Virtualization _Virtual Machine Management Guide_
+* link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Creating_a_template_from_an_existing_virtual_machine[Creating a Template] in the Red Hat Virtualization _Virtual Machine Management Guide_
+
+[id="machineset-creating-a-new-machine-set-on-the-new-storage-domain"]
+== Creating a new machine set on the new storage domain
+
+You can use the virtual machine template to add a new machine set to the {product-title} cluster.
+
+.Procedure
+
+. Get the details of the current machine set:
++
+[source,terminal]
+----
+$ oc get machineset -o yaml
+----
+
+. Use these details to create a machine set. For more information see _Creating a machine set_. Enter the new virtual machine template name in the *template_name* field. Use the same template name you used in the *Make template* dialog in the {rh-virtualization-engine-name}.
+
+. Note the names of both the old and new machine sets. You need to refer to them in subsequent steps.
+
+[id="machineset-migrating-workloads"]
+== Migrating workloads
+
+To migrate workloads, you gradually scale up the new machine set and then remove nodes from the old machine set. You stop the pods on the old machine and start them on the new machine.
+
+For details on manually scaling machine sets, see _Scaling a machine set manually_.
+
+.Procedure
+
+. Scale up the new machine set.
+
+. Stop the pods on the old machine. ???HOW???
+
+. Start the pods on the new machine. ???HOW???
+
+. Scale down the old machine set.
+
+. When the old machine set is scaled down to zero machines remove it.
```

Comment:
I think you can delete this step since it is documented in the next section

Reply:
Removed.